src/controllers/carpools.ts

```diff
@@ -0,0 +1,194 @@
+/// <reference path="../../typings/express-session/express-session.d.ts" />
```

Comment:
We already have a `carpools.ctrl.ts` file. Can you add these functions to that file instead? It seems weird to have 2 carpool controllers.